CIP-0036/test-vector.md

```diff
@@ -1,110 +1,135 @@
-# Test vector for CIP15
+# Test Vector for CIP-0036
 
-### Inputs
+## Keys
 
-Payment **public** key hex
+**Note:** Not all keys are required for certificate recreation.
+
+Payment **private** signing key cborHex:
 ```
-3273a5316e4de228863bd7cf8dac90d57149e1a595f3dd131073b84e35546676
+5820614fdfe13d403bee2014570b190d81851f17d8daca0b6dd1ce33014403191003
 ```
 
-Staking **private** key hex
+Payment **public** verification cborHex:
+```
+58207a24dd8e692cec94b612c2ec81f508aada96557c2052a447b9d197b006fa7d2a
+```
+
+Staking **private** signing key cborHex:
+```
+5820852fa5d17df3efdfdcd6dac53ec9fe5593f3c0bd7cadb3c2af76c7e15dfa8a5c
+```
+
+Staking **public** verification key cborHex:
+```
+5820e3cd2404c84de65f96918f18d5b445bcb933a7cda18eeded7945dd191e432369
+```
+Governance (Catalyst) **public** verification key cborHex:
 ```
-f5beaeff7932a4164d270afde7716067582412e8977e67986cd9b456fc082e3a
+58200036ef3e1f0d3f5989e2d155ea54bdb2a72c4c456ccb959af4c94868f473f5a0
 ```
 
-Catalyst **private** key hex
+Governance (Catalyst) **private** signing key cborHex:
 ```
 4820f7ce221e177c8eae2b2ee5c1f1581a0d88ca5c14329d8f2389e77a465655c27662621bfb99cb9445bf8114cc2a630afd2dd53bc88c08c5f2aed8e9c7cb89
```

Comment:
Not sure what this is but it ain't a key? Could it be a combined private key +  public key ?

Reply:
@KtorZ

Reply:
Let's change `**private**` to `**extended private**` then 👍